src/neo-vm/Types/InteropInterface.cs

```diff
@@ -17,11 +17,11 @@ public override byte[] GetByteArray()
     public class InteropInterface<T> : InteropInterface
         where T : class
     {
-        private T _object;
+        private readonly T _object;
```

Comment:
This is interesting. Means that Interop VM type is immutable. Agreed.
However... is it really true? Can't upper layers somehow DUP this type, and have side effects between them?

Reply:
Simple example:
Syscall.GetEnumerator
Dup
Invoke something using enumerator

Question is: will enumerator be readonly for the method that needs it? If not, will some "side effect" be allowed to happen between them?
I suspect we need it non-readonly for external access.